libgnucash/engine/Account.cpp

```diff
@@ -5094,14 +5061,8 @@ xaccAccountGetReconcileChildrenStatus(const Account *acc)
      * is found then we can assume not to include the children, that being
      * the default behaviour
      */
-    GValue v = G_VALUE_INIT;
-    gboolean retval;
     if (!acc) return FALSE;
-    qof_instance_get_path_kvp (QOF_INSTANCE (acc), &v,
-            {KEY_RECONCILE_INFO, KEY_INCLUDE_CHILDREN});
-    retval = G_VALUE_HOLDS_INT64 (&v) ? g_value_get_int64 (&v) : FALSE;
-    g_value_unset (&v);
-    return retval;
+    return boolean_from_key (acc, {KEY_RECONCILE_INFO, KEY_INCLUDE_CHILDREN});
```

Comment:
This refactor is not completely indentical to previous. It can return true if the kvp value has string "true"; previous code would only return true if the kvp value is a non-zero int. This side effect *should* be unimportant in practice.